contracts/test/shared/constants.t.sol

```diff
@@ -11,6 +11,11 @@ contract Constants {
   address MODERATOR = address(42_002);
   address USER = address(42_003);
 
+  address constant ADMIN = address(42_000);
+  address constant TREASURY = address(42_001);
+  address constant MODERATOR = address(42_002);
+  address constant USER = address(42_003);
```

Comment:
I don't imagine this change work, as there is already variable from the same name above, please make sure the tests are still working

Reply:
It's just because in my last tests I started writing "TREASURE" because that's how it was written in the Controlable.sol file, but otherwise it doesn't make any difference that they are written with two "S". 

Sorry for my english :)